lms/templates/courseware/courseware-chromeless.html

```diff
@@ -137,7 +137,7 @@
         var newHeight = contentElement.offsetHeight;
         var newWidth = contentElement.offsetWidth;
 
-        if (newWidth === lastWidth && newHeight === lastHeight) {
+        if (eventType !== 'load' && newWidth === lastWidth && newHeight === lastHeight) {
```

Comment:
In my testing prod and localhost, the `load` event is _always_ ignored, as the handler has already been fired by the `MutationObserver` with the same height/width values prior to the load completing.

It seems generally "correct" that we should always invoke the resize handler upon page load, regardless of previous mutation events.
We _could_ just always fire the handler on _any_ event, but the mutation fires ~2 dozen times, so that's probably overdoing it.